README.md

```diff
@@ -4,13 +4,87 @@
 
 This repo hosts the `community.kubernetes` Ansible Collection.
 
-As this is an independent Collection, it can be released on its own release cadence.
+The collection includes a variety of Ansible content to help automate the management of applications in Kubernetes and OpenShift clusters, as well as the provisioning and maintenance of clusters themselves.
 
-## Notes
+## Included content
 
-*WARNING* This is Work in progress
+Click on the name of a plugin or module to view that content's documentation:
 
-For more information see `#ansible-community` on Freenode IRC, or the [Kubernetes Working Group](https://github.com/ansible/community/wiki/Kubernetes) Community wiki page.
+  - **Connection Plugins**:
+    - [kubectl](https://docs.ansible.com/ansible/latest/plugins/connection/kubectl.html)
```

Comment:
Nice touch to list there here. Once the Collections docs are build we can update these link.